ci/doc/how-to-install-taos-ci.md

```diff
@@ -51,3 +51,8 @@ We will deliver event details when this hook is triggered.
 
 As a final step, press `Add webhook` button. That's all. 
 From now on, enjoy a CI world for more collaborative and productive software development!!!
+
+## Trouble shooting
```

Comment:
Could you replace "##" with "#"?
`s/##/#/g`

Reply:
Updated.